lib/matplotlib/colors.py

```diff
@@ -1650,20 +1650,23 @@ class BoundaryNorm(Normalize):
     Generate a colormap index based on discrete intervals.
 
     Unlike `Normalize` or `LogNorm`, `BoundaryNorm` maps values to integers
-    instead of to the interval 0-1.
-
-    Mapping to the 0-1 interval could have been done via piece-wise linear
-    interpolation, but using integers seems simpler, and reduces the number of
-    conversions back and forth between integer and floating point.
+    instead of to the interval 0-1.  (Mapping to the 0-1 interval could have
+    been done via piece-wise linear interpolation, but using integers seems
+    simpler, and reduces the number of conversions back and forth between int
+    and float.)
```

Comment:
Is out of scope of the PR, I know, but does the parenthetical part need to be rendered in the docs? This seems like an implementation detail?

Reply:
sure, moved it to a comment.